### DISCUSSION TITLE
哪吒监控Agent多开 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#41] [General]
https://blog.1l1.icu/2024/01/31/na-zha-jian-kong-duo-kai/? 

 复制原Agent服务配置1cp /etc/systemd/system/nezha.service /etc/systemd/system/<NEW-SERVICE>.service  编辑服务1vim /etc/systemd/system/<NEW-SERVICE>.service  编辑 ExecStart 部分若RPC启用了tls，还要在后面加上 --tls 123

**Comment by ltzXiaoYanMo** [comment]
:thinking:

~~没错我tm又来了（~~

**Comment by Muska-Ami** [comment]
> @ltzXiaoYanMo
> :thinking:
> 
> ~~没错我tm又来了（~~

谁让你来了（？）

**Comment by ltzXiaoYanMo** [comment]
> > @ltzXiaoYanMo
> > 🤔
> > ~没错我tm又来了（~
> 
> 谁让你来了（？）

不是你叫我来的吗（

**Comment by Muska-Ami** [comment]
@ltzXiaoYanMo 我草谁问你了(？)

**Comment by ltzXiaoYanMo** [comment]
> @ltzXiaoYanMo 我草谁问你了(？)

![image](https://github.com/Muska-Ami/muska-ami-blog/assets/115706851/87512cb5-46f0-4b40-acd9-baf0cf11aabb)

**Comment by Muska-Ami** [comment]
@ltzXiaoYanMo 看不见看不见（

**Comment by ltzXiaoYanMo** [comment]
> @ltzXiaoYanMo 看不见看不见（

没事，已经糊你脸上了（？）
